cypress/integration/router_spec.js

```diff
@@ -0,0 +1,278 @@
+import { c, a } from '../support/utils.js';
+
+describe('Create event', () => {
+  beforeEach(() => {
+    cy.resetDb();
+    cy.cachedLogin();
+  });
+
+  const openMenu = () => {
+    cy.get(c('buttonGroup')).within(() => {
+      cy.get(c('searchIcon')).click();
+    });
+  };
+
+  it('should be able to navigate to events', () => {
+    cy.visit('/');
+    cy.get(c('navigation')).within(() => {
+      cy.get(a('/events')).click();
+    });
+    cy.url().should('contain', '/events');
+    cy.contains('Denne uken');
+    cy.contains('Liste');
+  });
+
+  it('should be able to navigate to joblistings', () => {
+    cy.visit('/');
+    cy.get(c('navigation')).within(() => {
+      cy.get(a('/joblistings')).click();
```

Comment:
Maybe it's better to select the link by its text content rather than its URL? That's how a user would find the right link to click. And you already assert the URL further down.

If you agree, the same applies to the rest of the tests.

Reply:
I was dabbling with the same thought, what is the best way to select an element, using `.contians` on the text or `.get` on some property. I thought about what was least likely to change over time. It was just an effort to keep the test viable in future rewrites.

But you are right about the _"That's how a user would find the right link"_, so maybe the `.contains` way is better.

Reply:
I think if you change "big" things like how navigation works or what things are named, you should expect to have to fix some tests. I'm not sure exactly what is best though, it's a bit hard to choose which selectors to use. 

In their best practices they have some examples of what cypress thinks you should do: https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements

![image](https://user-images.githubusercontent.com/5272089/56170850-62b8bd80-5fe3-11e9-8252-c17141e8ff45.png)

Reply:
In the case of these tests, I think the wording in each link is also an important thing to test, as it is essential for the use of navigation. If the URLs are correct but the text is wrong, the tests should ideally fail

Reply:
You got a good point there